addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13313,6 +13313,8 @@ msgctxt "#20375"
 msgid "Unassign content"
 msgstr ""
 
+#: xbmc/video/GUIViewStateVideo.cpp
```

Comment:
Please add a verbal description to help our translators so they can get their job done.
I mean thise lines starting with `#.` ;-)

Reply:
done